backend/endpoints/rooms.py

```diff
@@ -62,3 +63,75 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+
+@router.put(
+    "/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    status_code=status.HTTP_200_OK,
+    responses={
+        400: {"detail": "the max number for a Group_DM is 9"},
+        401: {"detail": "member not an admin"},
+        403: {"detail": "DM room or not found"},
+    },
+)
+async def add_to_room(
+    data: AddToRoom,
+    org_id: str,
+    room_id: str,
+    member_id: str,
+    background_tasks: BackgroundTasks,
+):
+    """Adds a new member(s) to a room
+    Args:
+        data: a pydantic schema that defines the request params
+        org_id (str): A unique identifier of an organisation
+        room_id: A unique identifier of the room to be updated
+        member_id: A unique identifier of the member initiating the request
+
+    Returns:
+        HTTP_200_OK: member added
+    Raises:
+        HTTP_400_BAD_REQUEST: the max number for a Group_DM is 9
+        HTTP_401_UNAUTHORIZED: member not in room or not an admin
+        HTTP_403_FORBIDDEN: DM room or not found
+    """
+
+    DB = DataStorage(org_id)
+    new_member = data.dict().get("new_member")
+    room = await get_room(org_id=org_id, room_id=room_id)
+    member = room.get("room_members").get(str(member_id))
+
+    if room is None or room["room_type"] == RoomType.DM:
+        raise HTTPException(status_code=403, detail="DM room or not found")
```

Comment:
In the code, can we replace the numeric value of the status code with the convenience variables from` fastapi.status`. I think it will be more understandable.
What do you think?
```suggestion
        raise HTTPException(status_code=HTTP_403_FORBIDDEN, detail="DM room or not found")
```

Reply:
Okay, I was just trying to keep the characters in each line of code as few as possible